tests/apidocs/endpoints/organizations/test-org-users.py

```diff
@@ -1,6 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Could we just remove these everywhere first in a diff PR?

Reply:
Sure.